config/plugins/visualizations/annotate_image/src/script.js

```diff
@@ -437,11 +437,13 @@ window.bundleEntries.load = function (opt) {
         });
     };
 
+    const slash_cleanup = /(\/)+/g;
+    const safe_download_url = `${options.root}/${dataset.download_url}`.replace(slash_cleanup, "/");
```

Comment:
```suggestion
    const safe_download_url = `${options.root}${dataset.download_url}`;
```
or `${getAppRoot()}/${dataset.download_url}`, which we use throughout the app